icu4c/source/data/misc/icuver.txt

```diff
@@ -9,6 +9,6 @@
 // ***************************************************************************
 icuver:table(nofallback){
     CLDRVersion{"38"}
```

Comment:
Does the CLDRVersion need to be updated to 39? The comment at the top says this file is autogenerated now.

Reply:
I think this is being updated by Peter in the PR for the CLDR integration:
https://github.com/unicode-org/icu/pull/1551/files#diff-e9d921715e4b4503ff366116da481b94d216a494b31b287b7d19ddd3e4e97641

Reply:
Ah, thanks Jeff!